react/Video.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+
+import { VideoPlayer } from './interfaces'
+import VimeoPlayer from './players/VimeoPlayer'
+import HTML5Player from './players/HTML5Player'
+import YoutubePlayer from './players/YoutubePlayer'
+
+const Video: StorefrontFunctionComponent<VideoPlayer> = (props) => {
+  const { src, name, description } = props
+  const isVimeo = /vimeo/.test(src)
+  const isYoutube = /youtube|youtu.be/.test(src)
+
+  const renderStructuredData = () => {
+    return (
+      <script
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{
+          __html: `{
+          "@context": "http://schema.org",
+          "@type": "VideoObject",
+          "name": ${name},
+          "description": ${description},
+          "contentUrl": ${src}
```

Comment:
```suggestion
          "name": ${name},
          "description": ${description},
          "contentUrl": ${src}
```

Required properties are not being filled
![image](https://user-images.githubusercontent.com/284515/92599957-4d4fcd80-f281-11ea-98af-deec68e356ad.png)